packages/office-ui-fabric-react/src/components/MessageBar/MessageBar.base.tsx

```diff
@@ -132,7 +132,7 @@ export class MessageBarBase extends BaseComponent<IMessageBarProps, IMessageBarS
     const { theme } = this.props;
     return (
       <div style={{ background: theme!.semanticColors.bodyBackground }}>
-        <div className={this._classNames.root}>
+        <div className={this._classNames.root} role="region">
```

Comment:
I'm not sure this is correct.

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Region_role

> Every element with a region role should include a label that describes the purpose of the content in the region, preferably with an aria-labelledby referencing a visible header. If no visible appropriate header is present, aria-label should be used..

There doesn't seem to be any way to get a label here.

Reply:
updated